airbyte-cdk/python/airbyte_cdk/sources/utils/schema_helpers.py

```diff
@@ -189,6 +189,17 @@ def dict(self, *args: Any, **kwargs: Any) -> dict[str, Any]:
         kwargs["exclude_unset"] = True
         return super().dict(*args, **kwargs)
 
+    def is_limit_reached(self, records_counter: int) -> bool:
+        """
+        Check if record count reached limit set by internal config.
+        :records_counter - number of records already red
```

Comment:
I don't remember which Python doc parser we use but it feels like we are using `:param` often. Should this be `:param records_counter: number of records already red`